spec/factories/tube_factories.rb

```diff
@@ -116,10 +118,25 @@
       parents { [] }
       purpose { create :v2_purpose, name: purpose_name, uuid: purpose_uuid }
 
+      siblings_count { 0 }
+      sibling_default_state { 'passed' }
+      siblings do
```

Comment:
Ah so is one `sibling_tubes` created by default, then if there is to be any more, `siblings_count` will be specified to be > 0>

Reply:
Not the way I would have designed it, but it's mimicking the way it's done for the v1 factory.  If I don't mimic that, I need to change the tests to match and at that point, who knows if the behaviour is unchanged?  For what it's worth, no the first sibling isn't an actual sibling.  It's a reference to the tube itself.  So it counts itself as a sibling of itself.  The sibling count is actually the number of **_other_** tubes it is related to.